python/cugraph-pyg/cugraph_pyg/loader/__init__.py

```diff
@@ -10,3 +10,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from cugraph_pyg.utilities.api_tools import experimental_warning_wrapper
```

Comment:
Maybe just a FIXME, but if cugraph is a hard dependency now, can we just remove the cugraph_pyg copy of these utils and use cugraph's?

Reply:
fixed